test/remote/push_tests.js

```diff
@@ -0,0 +1,54 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+var tap = require('tap')
+var proxyquire = require('proxyquire')
+var uuid = require('uuid')
+
+var test = tap.test
+var P = require('../../lib/promise')
+var mockLog = require('../mocks').mockLog
+var mockUid = new Buffer('foo')
+
+var PushManager = require('../push_helper').PushManager
+
+var pushManager = new PushManager({
+  server: 'wss://push.services.mozilla.com/',
```

Comment:
Hmmm...so this test is actually talking to the production push server?  I like it as an integration test but I wonder if we can segregate it somehow so our unittests don't involve external infrastructure.